app/views/questions/index.html.haml

```diff
@@ -13,17 +13,19 @@
   %tbody
     - @questions.each do |question|
       %tr
+      - if is_instructor? or question.published?
```

Comment:
This should actually be before the `%tr`, and everything under it should be indented. (If you view the generated html, the output is not nested properly.)